schema/helpers/mask.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Mask",
+    "description": "Mask for layer content.",
+    "allOf": [
+        {
+            "properties": {
+                "inv": {
+                    "title": "Inverted",
+                    "type": "boolean",
+                    "default": false
+                },
+                "mode": {
+                    "title": "Mode",
+                    "$ref": "#/$defs/constants/mask-mode",
+                    "default": "a"
+                },
+                "o": {
+                    "title": "Opacity",
+                    "$ref": "#/$defs/properties/scalar-property"
+                },
+                "pt": {
+                    "title": "Shape",
+                    "$ref": "#/$defs/properties/bezier-property"
+                },
+                "x": {
+                    "title": "Expansion",
```

Comment:
Add description for expansion? Something about how this is processed would be useful too.

Reply:
Dropped expansion, it is poorly supported across players.